pages/account/logout.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <div />
+</template>
+
+<script>
+  export default {
```

Comment:
Use the minimal layout, and move the logout to middleware, by applying this patch:
```diff
diff --git a/pages/account/login.vue b/pages/account/login.vue
index 5a3c1e18..47e971f9 100644
--- a/pages/account/login.vue
+++ b/pages/account/login.vue
@@ -6,9 +6,8 @@
   export default {
     auth: false,
     layout: 'minimal',
-    mounted() {
-      this.$auth.loginWith('keycloak');
+    middleware({ $auth }) {
+      $auth.loginWith('keycloak');
     }
   };
 </script>
-
diff --git a/pages/account/logout.vue b/pages/account/logout.vue
index 499a9ba3..ab45ad05 100644
--- a/pages/account/logout.vue
+++ b/pages/account/logout.vue
@@ -5,10 +5,12 @@
 <script>
   export default {
     auth: true,
-    layout: 'default',
-    middleware: 'auth',
-    mounted() {
-      this.$auth.logout();
-    }
+    layout: 'minimal',
+    middleware: [
+      'auth',
+      ({ $auth }) => {
+        $auth.logout();
+      }
+    ]
   };
 </script>
```
(It also applies the same approach to the login.)

Reply:
I noticed that, when I do login & logout this way, when I try to access some pages (account page and logout page) unauthorized, the redirection is not working correctly (user is stuck on blank login and logout pages, without being redirected to keycloak-login and home page, respectively)

Reply:
@rwd Should I proceed with this patch anyway?

Reply:
@spyrosbek I see.

How about this instead:
```diff
diff --git a/pages/account/login.vue b/pages/account/login.vue
index 5a3c1e18..80d5eb66 100644
--- a/pages/account/login.vue
+++ b/pages/account/login.vue
@@ -6,9 +6,8 @@
   export default {
     auth: false,
     layout: 'minimal',
-    mounted() {
+    created() {
       this.$auth.loginWith('keycloak');
     }
   };
 </script>
-
diff --git a/pages/account/logout.vue b/pages/account/logout.vue
index 499a9ba3..ab45ad05 100644
--- a/pages/account/logout.vue
+++ b/pages/account/logout.vue
@@ -5,10 +5,12 @@
 <script>
   export default {
     auth: true,
-    layout: 'default',
-    middleware: 'auth',
-    mounted() {
-      this.$auth.logout();
-    }
+    layout: 'minimal',
+    middleware: [
+      'auth',
+      ({ $auth }) => {
+        $auth.logout();
+      }
+    ]
   };
 </script>
```